src/eko/kernels/valence_qed.py

```diff
@@ -101,5 +52,37 @@ def dispatcher(
             singlet EKO
     """
     if method in ["iterate-exact", "iterate-expanded"]:
-        return eko_iterate(gamma_valence, a1, a0, aem_list, nf, order, ev_op_iterations)
-    raise NotImplementedError('Only "iterate-exact" is implemented with QED')
+        return eko_iterate(
+            gamma_valence, a1, a0, aem_list, nf, order, ev_op_iterations, dim=2
+        )
+    if method == "perturbative-exact":
+        return eko_perturbative(
```

Comment:
Why is this perturbative, if exact is selected? Moreover: is the total valence sector any different from the singlet one in QED? Apart for the dimension (that could be passed by argument)

Reply:
If you see singlet.py the methods `perturbative-exact` and `perturbative-expanded` both use `eko_perturbative` but one with `is_exact=True` and the other with `False`.

Reply:
Sorry, my bad :)